test/integration/default/build/ec2.tf

```diff
@@ -197,3 +197,16 @@ resource "aws_security_group" "alpha" {
 output "ec2_security_group_alpha_group_id" {
   value = "${aws_security_group.alpha.id}"
 }
+
+#============================================================#
+#                      VPC Subnets
+#============================================================#
+
+resource "aws_subnet" "default" {
```

Comment:
What makes this "default"?  Is there a better name we could give it as a test fixture - "alpha", "subnet-01", etc?